.gitignore

```diff
@@ -17,6 +17,7 @@ __pycache__
 /.vscode
 
 # tools
-/.*_cache
+.*_cache
```

Comment:
why is this change needed?

Reply:
I think it was because it wasn't ignoring `mypy_cache` before

Reply:
just tested again, it was only the `mypy_cache` in the `tests` folder, so I'm removing this change